Controller/FormController.php

```diff
@@ -363,6 +363,10 @@ private function getApiEntity(Form $entity, string $locale): array
                 $fieldData['options'] = $fieldTranslation->getOptions();
             }
 
+            if ($fieldData['type'] === 'radioButtons' && empty($fieldData['options'])) {
```

Comment:
I think this should be genrally be fixed for all types the problem is that `options` does return an array instead of an object I think and so we need todo:

```php
if (empty($fieldData['options'])) {
     $fieldData['options'] = new stdClass(); // convert options to "{}"
}
```

Then this should also work for multi dropdown and multi checkboxes

Reply:
Correct @alexander-schranz , updated ;-)

Reply:
> @cirykpopeye Great thank you this fixes #244

I had to add a \ in front of stdClass, I updated it in a push below.